tdms/CMakeLists.txt

```diff
@@ -134,6 +134,17 @@ endif()
 
 
 # Install ---------------------------------------------------------------------
+if (BUILD_TESTING)
+    if (APPLE)
+        list(APPEND CMAKE_INSTALL_RPATH "@executable_path/../lib")
+    elseif (UNIX)
+        list(APPEND CMAKE_INSTALL_RPATH "$ORIGIN/../lib")
+    endif()
```

Comment:
Windows? Or is that out of scope for `BUILD_TESTING=ON`?

Reply:
Windows has no concept of rpath and libraries and executables are in the same directory anyway.